src/reducers/rootReducer.js

```diff
@@ -0,0 +1,66 @@
+const initState = {
+  post: [
```

Comment:
`post` and `todo` seem to be unused. Why are they in your initial state?

Reply:
clean up